integration-commons/src/main/java/org/symphonyoss/integration/logging/IntegrationBridgeCloudLoggerFactory.java

```diff
@@ -16,32 +16,18 @@
 
 package org.symphonyoss.integration.logging;
 
-import static com.symphony.atlas.config.SymphonyAtlas.ACCOUNT;
-import static com.symphony.atlas.config.SymphonyAtlas.SECRET;
-import static com.symphony.atlas.config.SymphonyAtlas.SYMPHONY_URL;
+import static org.symphonyoss.integration.logging.IntegrationBridgeKeyProvider.CONFIG_HARVESTER_URL;
 
-import com.symphony.config.ConfigurationException;
 import com.symphony.logging.ISymphonyCloudIdentifierProvider;
 import com.symphony.logging.ISymphonyLogger;
 import com.symphony.logging.ISymphonyOnPremKeyProvider;
 import com.symphony.logging.SymphonyLoggerFactory;
-import com.symphony.security.cache.InMemoryPersister;
-import com.symphony.security.clientsdk.client.AuthProvider;
-import com.symphony.security.clientsdk.client.ClientIdentifierFilter;
-import com.symphony.security.clientsdk.client.SymphonyClient;
-import com.symphony.security.clientsdk.client.SymphonyClientConfig;
-import com.symphony.security.clientsdk.client.impl.SymphonyClientFactory;
-import com.symphony.security.exceptions.SymphonyEncryptionException;
-import com.symphony.security.exceptions.SymphonyInputException;
-import com.symphony.security.exceptions.SymphonyNativeException;
-
-import com.google.common.annotations.VisibleForTesting;
-
-import java.io.IOException;
-import java.util.Properties;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
+import com.symphony.logging.appender.OnPremHttpsAppender;
+
+import org.apache.logging.log4j.LogManager;
```

Comment:
@dnsymphony I made some tests but I can't get the appenders using the SLF4J API. It happens because all loggers are wrapped by org.apache.log4j.Logger class and this class always returns null when you call the method getAppender() or an empty enumeration when you call the method getAllAppenders(). 
Also, I can't find logger context class similar to org.apache.logging.log4j.core.LoggerContext in SLF4J API.